GPU/Software/SoftGpu.cpp

```diff
@@ -1138,6 +1145,18 @@ void SoftGPU::FinishDeferred() {
 	drawEngine_->transformUnit.Flush("finish");
 }
 
+int SoftGPU::ListSync(int listid, int mode) {
+	// Take this as a queue that we need to finish drawing.
```

Comment:
cue, not queue :)

Reply:
D'oh, I type queue way more often than cue, this happens with several words heh.

-[Unknown]